practice/src/component/Button/Button.tsx

```diff
@@ -1,47 +1,29 @@
 import './Button.css';
 
 interface ButtonProps {
-    /**
-   * Button contents
-   */
-    label: string;
-  /**
-   * Is this the principal call to action on the page?
-   */
-  variant?: 'primary' | 'secondary';
-  /**
-   * What background color to use
-   */
-  backgroundColor?: string;
-  /**
-   * How large should the button be?
-   */
-  size?: 'small' | 'medium' | 'large';
-  /**
-   * Optional click handler
-   */
+  label?: string;
+  variant?: 'ghost' | 'outline';
+  borderColor?: string;
+  size?: 'small' | 'medium';
+  icon?: React.ReactNode;
   onClick?: () => void;
 }
 
-/**
- * Primary UI component for user interaction
- */
 export const Button = ({
-  variant = 'primary', // default to 'primary' if not specified
+  variant = 'ghost',
   size = 'medium',
-  backgroundColor,
+  borderColor,
   label,
-  ...props
+  icon,
 }: ButtonProps) => {
-  const mode = variant === 'primary' ? 'button--primary' : 'button--secondary';
+  const mode = variant === 'ghost' ? 'button--ghost' : 'button--outline';
   return (
     <button
```

Comment:
Have you added style for active button?
<img width="282" alt="Screenshot 2024-05-30 at 10 26 55" src="https://github.com/huynhthikimtrung321/react-training/assets/24991129/a2b5e347-f4c1-431c-b2a2-1b4ee3e70063">